internal/cmd/operator-sdk/cli/cli.go

```diff
@@ -19,56 +19,79 @@ import (
 	"sigs.k8s.io/kubebuilder/v3/pkg/cli"
-	cfgv2 "sigs.k8s.io/kubebuilder/v3/pkg/config/v2"
 	cfgv3 "sigs.k8s.io/kubebuilder/v3/pkg/config/v3"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
+	golangv2 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/v2"
+	golangv3 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/v3"
 
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/bundle"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/cleanup"
-	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/completion"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/generate"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/olm"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/run"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/scorecard"
 	"github.com/operator-framework/operator-sdk/internal/flags"
+	"github.com/operator-framework/operator-sdk/internal/plugins"
 	ansiblev1 "github.com/operator-framework/operator-sdk/internal/plugins/ansible/v1"
-	golangv2 "github.com/operator-framework/operator-sdk/internal/plugins/golang/v2"
-	golangv3 "github.com/operator-framework/operator-sdk/internal/plugins/golang/v3"
+	envtestv1 "github.com/operator-framework/operator-sdk/internal/plugins/envtest/v1"
 	helmv1 "github.com/operator-framework/operator-sdk/internal/plugins/helm/v1"
+	manifestsv2 "github.com/operator-framework/operator-sdk/internal/plugins/manifests/v2"
+	scorecardv2 "github.com/operator-framework/operator-sdk/internal/plugins/scorecard/v2"
 	"github.com/operator-framework/operator-sdk/internal/util/projutil"
 )
 
 var commands = []*cobra.Command{
 	bundle.NewCmd(),
 	cleanup.NewCmd(),
-	completion.NewCmd(),
 	generate.NewCmd(),
 	olm.NewCmd(),
 	run.NewCmd(),
 	scorecard.NewCmd(),
 }
 
 func Run() error {
-	cli, _ := GetPluginsCLIAndRoot()
-	return cli.Run()
+	c, _ := GetPluginsCLIAndRoot()
+	return c.Run()
 }
 
 // GetPluginsCLIAndRoot returns the plugins based CLI configured to use operator-sdk as the root command
 // This CLI can run kubebuilder commands and certain SDK specific commands that are aligned for
 // the kubebuilder project layout
 func GetPluginsCLIAndRoot() (cli.CLI, *cobra.Command) {
+	ansibleBundle, _ := plugin.NewBundle("ansible"+plugins.DefaultNameQualifier, plugin.Version{Number: 1},
+		ansiblev1.Plugin{},
+		manifestsv2.Plugin{},
+		scorecardv2.Plugin{},
+	)
+	gov2Bundle, _ := plugin.NewBundle(golangv2.Plugin{}.Name(), golangv2.Plugin{}.Version(),
+		golangv2.Plugin{},
+		envtestv1.Plugin{},
+		manifestsv2.Plugin{},
+		scorecardv2.Plugin{},
+	)
+	gov3Bundle, _ := plugin.NewBundle(golangv3.Plugin{}.Name(), golangv3.Plugin{}.Version(),
+		golangv3.Plugin{},
+		manifestsv2.Plugin{},
+		scorecardv2.Plugin{},
+	)
+	helmBundle, _ := plugin.NewBundle("helm"+plugins.DefaultNameQualifier, plugin.Version{Number: 1},
+		helmv1.Plugin{},
+		manifestsv2.Plugin{},
+		scorecardv2.Plugin{},
```

Comment:
IMO:We cannot have address this scenario as `go.sdk.operatorframework.io` because it is not the result that we want. We want SDK scaffolding KB plugins and doing things on top by default. 

However, that might be valid for others types (hybrid scenario go+helm+caveats). So, I am ok with this solution to not break the backwards compatibilities and bring the flexibilities for we starts to try to address this needs for example.  

However, we need to think how we tell for the CLI what are the plugins that should be scaffold soon top always? It is something that we might be able to address for what still in the 2.0 upstream issue as well with the requirement to find a way of the bin discovery the plugins.IMO:We cannot have address this scenario as `go.sdk.operatorframework.io` because it is not the result that we want. We want SDK scaffolding KB plugins and doing things on top by default. 

However, that might be valid for others types (hybrid scenario go+helm+caveats). So, I am ok with this solution to not break the backwards compatibilities and bring the flexibilities for we starts to try to address this needs for example.  

However, we need to think how we tell for the CLI what are the plugins that should be scaffold soon top always? It is something that we might be able to address for what still in the 2.0 upstream issue as well with the requirement to find a way of the bin discovery the plugins.